index.html

```diff
@@ -19,7 +19,7 @@
         shortName: "vc-data-model",
 
         // subtitle for the spec
-        subtitle: "Expressing verifiable information on the Web",
+        subtitle: "Expressing information with verifiable provenance",
```

Comment:
Another option:

```suggestion
        subtitle: "Expressing information with verifiable authorship",
```

Reply:
This one feels better... except that we don't talk about "authors" in the document anywhere.

Reply:
It's more than source and/or authorship ...there is an an aspect that deals with the content of the claims as well.

A VC is expressing an trustable attestation that a set of claims has not been modified relative to a point in time (issuance of the VC).

Perhaps, "Expressing a set of immutable attributes relative to a point in time".